webui/react/src/App.tsx

```diff
@@ -27,15 +31,26 @@ const AppView: React.FC = () => {
   const { isAuthenticated, user } = Auth.useStateContext();
   const cluster = ClusterOverview.useStateContext();
   const info = Info.useStateContext();
+  const setInfo = Info.useActionContext();
   const showSpinner = FullPageSpinner.useStateContext();
   const setShowSpinner = FullPageSpinner.useActionContext();
   const username = user ? user.username : undefined;
+  const [ infoResponse, requestInfo ] = useRestApiSimple<{}, DeterminedInfo>(getInfo, {});
+
+  const fetchInfo = useCallback(() => requestInfo({}), [ requestInfo ]);
 
   updateFaviconType(cluster.allocation !== 0);
 
   useRouteTracker();
   useTheme();
 
+  usePolling(fetchInfo, { delay: 1000 });
```

Comment:
suggestion: higher than default delay? I think /info response would only change when someone updates master or the cluster.

Reply:
great point, not sure why I had it so high!